shared/settings/chat/index.js

```diff
@@ -0,0 +1,198 @@
+              const wlremoved = this._isUnfurlWhitelistRemoved(w)
+              return (
+                <React.Fragment key={w}>
+                  <Kb.Box2
+                    fullWidth={true}
+                    direction="horizontal"
+                    style={Styles.collapseStyles([
+                      wlremoved ? {backgroundColor: Styles.globalColors.red_20} : undefined,
+                      styles.whitelistRowContainer,
+                    ])}
+                  >
+                    <Kb.Text type="BodySemibold">{w}</Kb.Text>
+                    <Kb.Text
+                      type="BodyPrimaryLink"
+                      style={wlremoved ? {color: Styles.globalColors.white} : null}
+                      onClick={() => this._toggleUnfurlWhitelist(w)}
+                    >
+                      {wlremoved ? 'Restore' : 'Remove'}
+                    </Kb.Text>
+                  </Kb.Box2>
+                  <Kb.Divider fullWidth={true} />
+                </React.Fragment>
+              )
+            })}
+          </Kb.ScrollView>
+          <Kb.RadioButton
+            key="rbnever"
+            label="Never"
+            onSelect={() => this._setUnfurlMode(RPCChatTypes.unfurlUnfurlMode.never)}
+            selected={this._getUnfurlMode() === RPCChatTypes.unfurlUnfurlMode.never}
+            disabled={this.props.unfurlMode === undefined}
+          />
+        </Kb.Box2>
+        <Kb.Divider style={styles.divider} />
+        <Kb.Box2 direction="vertical" gap="tiny">
+          <Kb.WaitingButton
+            onClick={() => this.props.onUnfurlSave(this._getUnfurlMode(), this._getUnfurlWhitelist(true))}
+            label="Save"
+            type="Primary"
+            style={styles.save}
+            disabled={this._isSaveDisabled()}
+            waitingKey={Constants.waitingKey}
+          />
+          {this.props.unfurlError && (
+            <Kb.Text type="BodySmall" style={styles.error}>
+              {this.props.unfurlError}
+            </Kb.Text>
+          )}
+        </Kb.Box2>
+      </Kb.Box2>
+    )
+  }
+}
+
+const styles = Styles.styleSheetCreate({
+  container: Styles.platformStyles({
+    isElectron: {
+      marginLeft: 28,
+      paddingTop: 20,
+    },
+    isMobile: {
+      padding: 20,
+    },
+  }),
+  divider: {
+    height: 2,
+  },
+  save: Styles.platformStyles({
+    common: {
+      marginTop: 8,
+    },
+  }),
+  whitelist: Styles.platformStyles({
+    common: {
+      alignSelf: 'flex-start',
+      borderWidth: 1,
+      borderColor: Styles.globalColors.lightGrey,
+      borderRadius: Styles.borderRadius,
+      borderStyle: 'solid',
+    },
+    isElectron: {
+      height: 150,
+      minWidth: 305,
+      marginLeft: 26,
+    },
+    isMobile: {
+      height: 150,
+      width: '100%',
+    },
+  }),
+  whitelistRowContainer: {
+    flexShrink: 0,
+    justifyContent: 'space-between',
+    paddingTop: 3,
+    paddingBottom: 4,
+    paddingLeft: 9,
+    paddingRight: 8,
```

Comment:
In general, it feels bad to have magic number pixel offsets like these -- it'd be nicer to use `Styles.globalMargins.` values instead, so that any house style design changes in the future affect this screen too.

Reply:
Yeah agreed, sometimes Zeplin just has some random px value in there so I am not sure how to best represent here. Will change these though.